layers/+completion/auto-completion/packages.el

```diff
@@ -270,7 +267,7 @@
 
 (defun auto-completion/init-yasnippet ()
   (use-package yasnippet
-    :commands (yas-global-mode yas-minor-mode yas-activate-extra-mode)
+    :commands (yas-global-mode yas-minor-mode yas-activate-extra-mode yas-hippie-try-expand)
```

Comment:
FYI :commands will make the autoloaded function "interactive" so it will show up in M-x, but this is wrong because it has no interactive form and will fail to be invoked this way.